db/migrate/20250725050517_create_ticket_log_changes.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class CreateTicketLogChanges < ActiveRecord::Migration[7.2]
+  def change
+    reversible do |dir|
+      dir.up do
+        create_table :ticket_log_changes do |t|
+          t.references :ticket_log, null: false
+          t.string :field_name, null: false
+          t.string :field_value, null: false
+          t.timestamps
+        end
+
+        TicketLog.find_each do |ticket_log|
+          if ticket_log.update_status?
+            ticket_log.ticket_log_changes.create!(
+              field_name: TicketLogChange.field_names[:status],
+              field_value: ticket_log.action_value,
+            )
+          end
+        end
+      end
+
+      dir.down do
+        drop_table :ticket_log_changes
+      end
```

Comment:
Unnecessary if you move the `create_table` out of the whole `reversible` block, per my comment above.

Reply:
Okay, agreed. Changed.